src/main/java/ch/njol/skript/events/EvtPlayerStatisticIncrease.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.bukkitutil.EntityUtils;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.EventValues;
+import org.bukkit.Statistic;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerStatisticIncrementEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Locale;
+
+public class EvtPlayerStatisticIncrease extends SkriptEvent {
+
+	static {
+		Skript.registerEvent("Player Statistic Increase", EvtPlayerStatisticIncrease.class, PlayerStatisticIncrementEvent.class,
+			"player statistic increase [of %*-strings%]")
```

Comment:
```suggestion
			"player statistic increase [of %-strings%]")
```
Just realized after looking in skriptlang-development
But you dont need the `*` (literal identifier) here because it can only ever be a literal.